.github/workflows/ci.yml

```diff
@@ -1,35 +0,0 @@
-name: Continuous integration
-
-on: [push]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Git checkout
-      uses: actions/checkout@v1
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.7
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip poetry
-        poetry install
-    - name: Lint with flake8 and Black
-      run: |
-        poetry run flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        poetry run flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --statistics
-        poetry run black --check .
-        poetry run mypy --ignore-missing-imports --disallow-untyped-defs .
-
-    - name: Test with pytest
-      run: |
-        poetry run pytest --cov=./ --cov-report=xml
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1.0.0
-      with:
-        token: ${{secrets.CODECOV_TOKEN}}
-        file: ./coverage.xml
-        flags: unittests
```

Comment:
@vhemt Is this line important? I can't seem to recreate the behaviour in codecov with their configuration file 🤔

Reply:
I have no idea, I think I just copied and pasted it :)

Reply:
Let's remove it then 👍